### PR TITLE
Feature/us1502 view a post

### PR DESCRIPTION
**Pull request by charlie-retzler**
Please review both crds-corkboard and crds-angular pull requests under Feature/US1502 view a post

### CODE REVIEW

File: Gateway/CorkboardAPI/CorkboardAPI/Web.config [hunk 1]
```diff
@@ -13,7 +13,7 @@
   </connectionStrings>
   <appSettings>
     <!--<add key="mongoConnectionString" value="mongodb://localhost:27017/local" />-->
-    <add key="mongoConnectionString" value="mongodb://localhost:27017/corkboard" />
+    <add key="mongoConnectionString" value="mongodb://int.crossroads.net:27017/corkboard" />
```
Comment (IngageBrewerM) [inline Gateway/CorkboardAPI/CorkboardAPI/Web.config]: should this be in a web.config transform that runs on deployment?

File: website/app/corkboard/corkboard-listings.html [hunk 2]
```diff
@@ -4,156 +4,47 @@
...
       <div class="col-xs-12 col-sm-6 col-md-4 corkboard-browse">
-
         <h4 class="flush-top">Browse</h4>
-
-          <a href="" class="btn btn-gray block btn-labeled push-half-bottom">Needs Posted
-            <span class="btn-label corkboard-need">
-            <svg viewBox="0 0 32 32" class="icon icon-lifebuoy">
-              <use xlink:href="#lifebuoy"></use>
+        <a ng-repeat="postType in corkboard.postTypeList"
+           ui-sref="{{corkboard.params.type === postType.filterName ? 'corkboard.base' : 'corkboard.filtered({type: postType.filterName})'}}"
```
Comment (IngageBrewerM) [inline website/app/corkboard/corkboard-listings.html]: Possibly move to a method in the controller with a descriptive name about what it is doing?

File: website/app/corkboard/corkboard-listings.html [hunk 3]
```diff
@@ -4,156 +4,47 @@
...
-
       <div class="col-xs-12 col-sm-6 col-md-4 corkboard-browse">
-
         <h4 class="flush-top">Browse</h4>
-
-          <a href="" class="btn btn-gray block btn-labeled push-half-bottom">Needs Posted
-            <span class="btn-label corkboard-need">
-            <svg viewBox="0 0 32 32" class="icon icon-lifebuoy">
-              <use xlink:href="#lifebuoy"></use>
+        <a ng-repeat="postType in corkboard.postTypeList"
```
Comment (IngageBrewerM) [inline website/app/corkboard/corkboard-listings.html]: Possibly move to a method in the controller with a descriptive name about what it is doing?

File: website/app/corkboard/corkboard-listings.html [hunk 4]
```diff
@@ -4,156 +4,47 @@
...
+           ui-sref="{{corkboard.params.type === postType.filterName ? 'corkboard.base' : 'corkboard.filtered({type: postType.filterName})'}}"
+           class="btn btn-gray block btn-labeled push-half-bottom"
```
Comment (IngageBrewerM) [inline website/app/corkboard/corkboard-listings.html]: Possibly move to a method in the controller with a descriptive name about what it is doing?

File: website/app/corkboard/corkboard-listings.html (hunk 2 above)
Comment (IngageBrewerM) [inline website/app/corkboard/corkboard-listings.html]: Possibly move to a method in the controller with a descriptive name about what it is doing?

File: Gateway/CorkboardAPI/CorkboardAPI/Web.config (hunk 1 above)
Comment (charlie-retzler) [inline Gateway/CorkboardAPI/CorkboardAPI/Web.config]: The default is for developers to work with the integration DB and not locally. The values for integration and demo environments are part of the web.config transforms and deploy process.

**Comment by charlie-retzler** [comment]
@IngageBrewerM Remerged development into US1502 but no changes needed. The merge that had been done was "lost" when I previously did a merge from development, then stashed changes, and then committed to the branch.

**Comment by IngageBrewerM** [comment]
:+1:

**Pull request merged**